tests/tracing/test_tracing_client.py

```diff
@@ -132,3 +136,29 @@ def test_tracing_client_calculate_trace_filter_correlation_without_base_filter()
         assert result.filter2_count == 0
         assert result.joint_count == 0
         assert result.total_count == 100
+
+
+@pytest.mark.parametrize("sql_warehouse_id", [None, "some-warehouse-id"])
+def test_tracing_client_search_traces_with_model_id(monkeypatch, sql_warehouse_id):
+    if sql_warehouse_id:
+        monkeypatch.setenv(MLFLOW_TRACING_SQL_WAREHOUSE_ID.name, sql_warehouse_id)
+    else:
+        monkeypatch.delenv(MLFLOW_TRACING_SQL_WAREHOUSE_ID.name, raising=False)
+    mock_store = Mock()
+    mock_store.search_traces.return_value = ([], None)
+
+    with patch("mlflow.tracing.client._get_store", return_value=mock_store):
```

Comment:
do we need to mock the entire store?

Reply:
I wanted to test no matter what store it is search_traces is invoked with these params, what's your proposal?